vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoDarkTheme.java

```diff
@@ -164,10 +162,6 @@ public LumoDarkTheme() {
         setDataLabelsDefaults(getPlotOptions().getPyramid().getDataLabels());
         setDataLabelsDefaults(getPlotOptions().getSpline().getDataLabels());
         setDataLabelsDefaults(getPlotOptions().getWaterfall().getDataLabels());
-
-        States states = new States();
-        states.setHover(new Hover(false));
-        getPlotOptions().getPie().setStates(states);
```

Comment:
This was added initially with https://github.com/vaadin/vaadin-charts-flow/pull/393. From the `Hover` JavaDoc I can not quite understand what the `enabled` state is supposed to do, but apparently disabling it doesn't do anything sensible it then marks all chart slices as inactive on hover. Removing this configuration seems to make the chart behave like any other without a theme.